assignment/assignment.hs

```diff
@@ -30,19 +31,22 @@ factorial n = 0 -- replace me
 --
 -- You may also need list concatenation, `++`.
 
-toDigits :: Int -> [Char]
-toDigits num = "replace me"
-
-
+toDigits :: Int -> [Char] -- think of the base case
+toDigits 0 = []        -- is this a sufficent placeholder? Doesn't account for 0 tho          
+toDigits num = toDigits (result) ++ [intToDigit(remainder)] --building from ones to tens to hundreds...
+    where (result, remainder) = divMod num 10           --tried using fst and snd o selected the truncated Result or the remainder, but ran into errors, tried different approach
+                                                              -- and worked
```

Comment:
That's a nice answer. In general, pattern matching the way you have it is clearer than poking around with fst and snd.